_shows/16_17/escape_for_dummies_edin.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Escape for Dummies"
+devised: true 
+period: Edinburgh
+season: Edinburgh
+season_sort: 590
+date_start: 2017-08-05
+date_end: 2017-08-25
+venue: Royal Terrace, Greenside
+venue_sort: Greenside 
+
+assets:
+  - type: poster
+    image: NfnQCWk
+
+links:
+  - type: Review
+    href: http://edinburgh.fringeguru.com/reviews/edinburgh-2017/escape-for-dummies
+    snapshot: Ju6HL
+    publisher: FringeGuru
+    author: Richard Stamp
+    title: "Escape for Dummies"
+    date: 2017-08-19
+    stars: 5
+    quote: "Escape For Dummies is an hour of pure joy – hilarious and giddy, yet with characters who make you care. Plan ahead for this one, because it's performing on odd-numbered dates only… and you'd have to be a dummy to miss it."
+  - type: Review
+    href: https://thepanoptic.co.uk/2017/08/20/escape-for-dummies/
+    snapshot: 8kYeA
+    publisher: The Panoptic
+    author: Tara Carlin
+    title: "Escape for Dummies"
+    date: 2017-08-20
+    stars: 4
+    quote: "This play is a unique and hidden gem at the Fringe. It’s a rare show in that it is suitable for the entire family, as well as for deaf audience members."
+  - type: Review
+    href: http://edfringereview.com/r/WYh34KJATEyhgk0jXDx6yg
+    snapshot: 5Eb4u
+    publisher: edfringereview
+    author: James Tibbles
+    title: "Escape for Dummies"
+    date: 2017-08-20
+    stars: 5
+    quote: "Escape for Dummies comprises an ensemble of five who all embody distinct caricatures in a masterclass of physical comedy. A simple escape plot sees two manikins come to life around a myriad of silent stunts and scenes that has the audience laughing from the very first moment."
```

Comment:
Should we add a `[sic]` after `manikins`, as this is slightly different to the mannequins the actors actually play?